### ISSUE TITLE
Save state hotkey doesn't work

### ISSUE THREAD

**Issue by georgiptr** [schellingb/dosbox-pure#36]
I compiled the DOSBox Pure from the latest main branch on my Raspberry Pi 4, running the latest RetroPie on Raspbian. Everything works great except that this is the only core that somehow doesn't honor the save game shortcut (Select + Right Shoulder). All other cores I'm using are working correctly. Loading shortcut (Select + Left Shoulder) also works fine. 

I also wanted to express my warm feelings towards your contribution. Playing DOS games is finally very convenient, accessible, super easy and the integration with RetroArch unlocks awesome features! Thank you from my heart! I looked at what you did and I'm speechless. What you've done is not just coding. It's art!

**Comment by keithbowes** [comment]
> I looked at what you did and I'm speechless.

Yeah, I was trying DOS cores to see how they'd handle logical keyboard layouts, as RetroArch itself uses physical keyboard layouts (older versions would use logical layouts in X (but still physical layouts in KMS and Wayland), but newer versions won't).  I initially tried using an old DOS version of the Lynx web browser in dosbox-libretro, but that wouldn't work at all.  The games I downloaded from the links provided didn't work either (e.g. Gobman wouldn't completely start).

I was then looking into other DOS cores and found this one.  It works so much better than dosbox-libretro; however, Gobman doesn't work 100% in dosbox-pure either (in particular, the key I've assigned to right in RetroArch doesn't work, so I had to remap to assign another unused key (the key for the Y button) for that function).  I don't know why the libretro guys have multiple versions of DOSBox cores without any real effort to differentiate them.

BTW, saving states seems to work for me (Linux x86_64).

(BTW, I can't get Lynx to work in dosbox-pure either; I get a NO PACKET DRIVER error.  That's not important, though, as I have the native Linux version.)
